config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   get 'welcome/index'
```

Comment:
Why are you using this and not the rails syntax? `resources :welcome, only: [:index]`

Reply:
No special reason, just my own ignorance. :-) I forgot how to do this so I googled it. I can change this to the rails syntax

Reply:
No worries. To be fair, there are _definitely_ situations where it makes sense to not use the rails syntax. Sometimes it doesn't support the route you need and sometimes it's much more readable if you're doing some funky naming or redirecting.

In this case though, you have a controller, and a very common action for which you're trying to set up a route. Also, if for some reason you needed another welcome action, you would be able to just add to the `only: [ ]` array. Go ahead and make that change then merge! #approved